R/options.R

```diff
@@ -60,5 +60,9 @@ setMSnbaseFastLoad <- function(opt = TRUE) {
 
 ##' @rdname MSnbaseOptions
 isMSnbaseFastLoad <- function() {
-    MSnbaseOptions()$fastLoad
+    fast_load <- MSnbaseOptions()$fastLoad
+    ## For some odd reasons we get also NULL back - parallel processing?
+    if (!length(fast_load))
+        fast_load <- FALSE
+    fast_load
 }
```

Comment:
I also thought so. But I got very strange errors that took me one day to hunt down: seems that sometimes `MSnbaseOptions()$fastload` is simply not set and `NULL` is returned. This was somehow related to parallel processing.

Re: `camelCase`, I know, I'm inconsistent here. I used to use always only camelCase, but now I got used to snake case because I find it reads better - I can change back if you don't like.